packages/field-base/test/slot-styles-mixin.test.js

```diff
@@ -37,14 +40,19 @@ describe('slot-styles-mixin', () => {
           );
         }
       },
-    );
-  });
+  );
+
+  let wrapper, element, button;
 
-  beforeEach(() => {
+  beforeEach(async () => {
     wrapper = fixtureSync('<div></div');
     wrapper.attachShadow({ mode: 'open' });
-    element = document.createElement('slot-styles-mixin-element');
+    wrapper.shadowRoot.innerHTML = '<slot></slot>';
```

Comment:
This is the actual fix: Firefox didn't render the element after moving, because there was no `<slot>` element.